### PR TITLE
Add two new professions / Add the Combat Medic prof to military scenarios

### PR DESCRIPTION
**Pull request by Fallout48**
#### Summary

SUMMARY: Content "Adds the Relief Volunteer and Speed King/Queen professions. Adds the Combat Medic profession to Helicopter Crash and Overrun scenarios"

#### Purpose of change

I forgot to add the Combat Medic profession to the military scenarios because I'm a big dummy. But I also thought of two new professions while I was doing that. I hope it's not too out of scope for the PR.

#### Describe the solution

Adds the Combat Medic Profession to the Helicopter Crash and Overrun scenarios. Change their first-aid-kit to instead spawn with nexus's cool new military aid kit.

Adds two new professions:

_Relief Volunteer_ | **1**
Very basic first aid profession.  Starts with a little storage, some band-aids and bandages. Granola bars, and some bottles of water. Makes for a nice friend, or a happy stomach.

_Speed King / Speed Queen_ | **5**
Starts with motorcycle gear, a superbike, and 6 levels of driving. I mostly just wanted to have a profession that had all of the motorcycle clothes. Pretty decent armor and you can be very speedy right off the bat.

#### Describe alternatives you've considered

Leaving someone else to fix the combat medic scenario thing another time. Forever plaguing me with guilt and shame for my failure.
Not adding more professions.
Going to sleep.

#### Testing

Have sneakers always been so encumbering? I could have sworn they used to only have like 5-6 encumbrance total when worn with socks. But when I tried out the Relief Volunteer the feet encumbrance was 11. Might just be me.

Spawned in the evac shelter. Gorged myself on frozen granola bars. Died happy.
Spawned in the evac shelter. Superbike spawned. Crashed into a tree at max speed and lived.
Combat Medic successfully spawns in crashed helicopter and military base.

#### Additional context

We should look into more ways of sorting professions eventually. As more people and mods add more and more professions, we'll probably end up scrolling for a while to find the one we want. Same with the trait menus but that's not as bad.
1prof
2prof
I updated it after I took that ^ screenshot, but the Speed Boi now starts with motorcycle armor & boots, and he's wearing compression clothes instead of a tshirt and jeans.

### CODE REVIEW

**Comment by Fallout48** [comment]
I'm not sure what's going on with the JSON style check. I've put the professions.json file through the linter several times since it says it needs linting. But every time it says there's no new changes to show, and the style check fails.

File: data/json/professions.json [hunk 1]
```diff
@@ -4251,5 +4251,62 @@
...
+        "items": [ 
+          "helmet_motor", 
+          "touring_suit", 
+          "under_armor", 
+          "under_armor_shorts", 
+          "socks", 
+          "motorbike_boots", 
+          "gloves_leather", 
+          "wristwatch", 
+          "motorbike_armor"
```
Comment (AMurkin) [inline data/json/professions.json:4304]:
```suggestion
          "helmet_motor",
          "touring_suit",
          "under_armor",
          "under_armor_shorts",
          "socks",
          "motorbike_boots",
          "gloves_leather",
          "wristwatch",
          "motorbike_armor"
```

File: data/json/professions.json [hunk 2]
```diff
@@ -4251,5 +4251,62 @@
...
+    "type": "profession",
+    "ident": "super_biker",
+    "name": { "male": "Speed King", "female": "Speed Queen" },
+    "description": "You used to rule the roads before they were filled with wrecks, undead, and unspeakable horrors.  At least there's no speed limits in the apocalypse!",
+    "points": 5,
+    "skills": [ { "level": 6, "name": "driving" } ],
+    "vehicle": "superbike",
+    "items": {
+      "both": {
+        "items": [
```
Comment (AMurkin) [inline data/json/professions.json:4295]:
```suggestion
        "items": [
```

**Comment by Fallout48** [comment]
@AMurkin You're a legend!

**Pull request merged**